expression/builtin_cast_vec_test.go

```diff
@@ -33,6 +34,16 @@ var vecBuiltinCastCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETJson}},
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDatetime}},
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDuration}},
+		{
+			retEvalType:   types.ETInt,
+			childrenTypes: []types.EvalType{types.ETString},
+			geners:        []dataGenerator{&numStrGener{rangeInt64Gener{math.MinInt64 + 1, 0}}},
+		},
+		{
+			retEvalType:   types.ETInt,
+			childrenTypes: []types.EvalType{types.ETString},
+			geners:        []dataGenerator{&numStrGener{rangeInt64Gener{0, math.MaxInt64}}},
+		},
```

Comment:
Can we combine these two test cases?

Reply:
Negative, it will certainly cause the `rangeInt64Gener` to overflow since it computes the interval length first.
Actually, I think we shouldn't wrap a `dataGenerator` interface instead of `rangeInt64Gener` into `numStrGener`. So we can use `defaultGener`.